.golangci.yml

```diff
@@ -13,16 +13,16 @@
 #
 
 run:
-  concurrency: 4
-  timeout: 5m
-  issues-exit-code: 0 # TODO: once all enabled linters are satisfied  
+  concurrency: 8
+  timeout: 2m
   tests: false
   skip-dirs:
     - ^experimental.*($|/)
+    - ^vendor.*($|/)
```

Comment:
*vendor* directories are already skipped when `skip-dirs-use-default` is enabled.